packages/frontend/src/routes/WalletWrapper.js

```diff
@@ -28,7 +28,7 @@ export function WalletWrapper({
     const createFromImplicitSuccess = useSelector(selectCreateFromImplicitSuccess);
     const createCustomName = useSelector(selectCreateCustomName);
     const fungibleTokensList = useFungibleTokensIncludingNEAR();
-    const tokensLoader = useSelector((state) => selectTokensLoading(state, { accountId })) || !balance?.total;
+    const tokensLoader = useSelector((state) => selectTokensLoading(state, { accountId }));
```

Comment:
Probably out of scope for this PR but should this be `tokensLoading` instead?

Reply:
Great catch! Fixed it here: https://github.com/near/near-wallet/pull/2563/commits/71f4692d487ce9f5cd165498e2b0e2f073f997f4